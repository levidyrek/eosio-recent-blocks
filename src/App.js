import React from 'react';
import './App.css';

import RecentBlocks from './components/RecentBlocks';

function App() {
  return (
    <div className="App">
      <h1>EOSIO Recent Blocks</h1>
      <RecentBlocks />
    </div>
  );
}

export default App;
