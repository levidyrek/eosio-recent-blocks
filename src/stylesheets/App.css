.App {
  margin-top: 20px;
}

.logo, .title {
  display: inline-block;
}

.logo {
  vertical-align: middle;
}
