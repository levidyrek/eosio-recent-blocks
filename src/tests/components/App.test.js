/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../App';

jest.mock('../../containers/RecentBlocks', () => () => <div className="RecentBlocks" />);

test('Render component', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
