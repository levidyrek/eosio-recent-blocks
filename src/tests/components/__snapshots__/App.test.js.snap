// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component 1`] = `
<div
  className="App"
>
  <h1>
    EOSIO Recent Blocks
  </h1>
  <div
    className="RecentBlocks"
  />
</div>
`;
